web/jsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerOptions": {
+        "baseUrl": "./app",

```

Comment:
This file is for vscode to support absolute paths.
Not sure we should it keep in repo.
Tell me what you think about it.

Reply:
It looks like a regular editor's env file, which need to be added to gitignore.